python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -32,8 +32,9 @@ def __init__(self, globalTag, jesUncertainties = [ "Total" ], jetType = "AK4PFch
         # read jet energy scale (JES) uncertainties
         self.jesInputFilePath = os.environ['CMSSW_BASE'] + "/src/PhysicsTools/NanoAODTools/data/jme/"
         if len(jesUncertainties) == 1 and jesUncertainties[0] == "Total":
-            self.jesUncertaintyInputFileName = "Summer16_23Sep2016V4_MC_Uncertainty_AK4PFchs.txt"
+            self.jesUncertaintyInputFileName = "Fall17_17Nov2017_V4_MC_Uncertainty_AK4PFchs.txt"
```

Comment:
we cannot do it this way, we must support 2016 and 2017 (same as for b-tag)

btw please note that there is a PR from Sal restructuring a bit this stuff... so I'm not sure if this is going to conflict